### PR TITLE
Expert Filter : Auto complete disabled  on field selection + cursor c…

### PR DESCRIPTION
**Pull request by basseche**

### CODE REVIEW

File: src/components/inputs/reactQueryBuilder/FieldSelector.tsx [hunk 1]
```diff
@@ -32,7 +32,13 @@ export function FieldSelector({
             disableClearable
             size="small"
             title={title}
-            renderInput={(params) => <TextField {...params} label="" variant="standard" />}
+            renderInput={({ inputProps, ...rest }) => (
```
Comment (ghazwarhili) [inline src/components/inputs/reactQueryBuilder/FieldSelector.tsx:35]: why this ...rest ?

**Review by Mathieu-Deharbe — COMMENTED** [review]
OK but I am a bit annoyed by the fact that you can't select anything anymore with the keyboard.

Like for the dropbox on the right side : 

sélection clavier

**Comment by basseche** [comment]
> OK but I am a bit annoyed by the fact that you can't select anything anymore with the keyboard.
> 
> Like for the dropbox on the right side : I select the dropbox with "tab" and then press '=' to select the right line :
> 
> sélection clavier [ sélection clavier ](https://private-user-images.githubusercontent.com/148252167/381124168-d76d1e26-5c9d-4528-b65c-761b8d80cbd3.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyMjM5NTQsIm5iZiI6MTczMDIyMzY1NCwicGF0aCI6Ii8xNDgyNTIxNjcvMzgxMTI0MTY4LWQ3NmQxZTI2LTVjOWQtNDUyOC1iNjVjLTc2MWI4ZDgwY2JkMy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOVQxNzQwNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTdhNTI5MGJkOWY4NWI0Yjg1Mzc5Zjc3MzM5NDE1OGJhZTJiYzI2Nzg5Y2QxMjNkNDExYzMwZGUzMDJiZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.MHElziBSgYPnjl4nHU-PcEbMVqS8JeCcwmac2IlLr7Q) [ ](https://private-user-images.githubusercontent.com/148252167/381124168-d76d1e26-5c9d-4528-b65c-761b8d80cbd3.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyMjM5NTQsIm5iZiI6MTczMDIyMzY1NCwicGF0aCI6Ii8xNDgyNTIxNjcvMzgxMTI0MTY4LWQ3NmQxZTI2LTVjOWQtNDUyOC1iNjVjLTc2MWI4ZDgwY2JkMy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOVQxNzQwNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTdhNTI5MGJkOWY4NWI0Yjg1Mzc5Zjc3MzM5NDE1OGJhZTJiYzI2Nzg5Y2QxMjNkNDExYzMwZGUzMDJiZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.MHElziBSgYPnjl4nHU-PcEbMVqS8JeCcwmac2IlLr7Q)

Ok Solved

**Comment by basseche** [comment]
@Mathieu-Deharbe : changed i put the old component with a css to control list height

**Review by thangqp — APPROVED** [review]

**Review by Mathieu-Deharbe — APPROVED** [review]
Small comment, but I don't have any better idea and it works.

**Pull request merged**